style/js/shield.js

```diff
@@ -242,9 +237,14 @@ function generateShieldCtx(id) {
     return ShieldDraw.blank();
   }
 
+  // Handle special case for Kentucky
```

Comment:
Kentucky is the only place that's taking advantage of this right now, but certainly other route networks will need this special casing too.

Reply:
That's terrifying.

Reply:
Off the top of my head, HCTRA and some New York State Parkways are gonna need this treatment.